components/admin_console/request_button/request_button.jsx

```diff
@@ -177,7 +177,7 @@ export default class RequestButton extends React.Component {
                 },
                 (err) => {
                     let errMsg = err.message;
-                    if (this.props.includeDetailedError) {
+                    if (this.props.includeDetailedError && err.detailed_error) {
```

Comment:
I'm not sure this will introduce a breaking change, I don't think so because if `detailed_error` is falsy, it'll return ` - ` at the end of the error.

Reply:
@threepwood-mm - can you elaborate on this? What do you mean by "it'll return `-` at the end of the error" ?

Reply:
yes, if `includeDetailedError` is true and `detailed_error` is not defined, it'll add ` - undefined`. (Sorry I said ` - `)

Reply:
ah. got it. `- undefined` is not pretty and I think should be handled.